app/authconfig_data.go

```diff
@@ -64,3 +69,15 @@ func addAuthConfig(name, aType string, enabled bool, management *config.Manageme
 
 	return nil
 }
+
+func createMgmtNamespace(management *config.ManagementContext) error {
+	_, err := management.Core.Namespaces("").Create(&corev1.Namespace{
```

Comment:
shall we try fetching the namespace first from cache? and crate only when doesn't exist

Reply:
This is the first time when auth configs get registered, where namespace would get created for first time, so thought didn't need to list first